libs/localization/src/lib/en.json

```diff
@@ -82,7 +82,7 @@
       "Unknown": "unknown"
     },
     "ShiftCohort": {
```

Comment:
should this be updated as well?
```
"ShiftCohort": {
```

Reply:
Yeah I was thinking about updating the key as well. but looks like we have components named by ShiftCohort, and there are too many changes if we want to rename all to Switch cohort. Since it doesn't impact user's experience or code readability if we continue using ShiftCohort as key in codespace, I only updated the localization strings here.
![image](https://user-images.githubusercontent.com/91754176/163048563-ae5b1d6b-c1f1-499d-bd0f-c3c708fdbd66.png)